cluster-state-service/handler/api/v1/taskapis.go

```diff
@@ -24,19 +24,22 @@ import (
 	"github.com/blox/blox/cluster-state-service/handler/store"
 	"github.com/blox/blox/cluster-state-service/handler/types"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 )
 
 const (
 	taskARNKey     = "arn"
 	taskClusterKey = "cluster"
 
-	taskStatusFilter  = "status"
-	taskClusterFilter = "cluster"
+	taskStatusFilter    = "status"
+	taskClusterFilter   = "cluster"
+	taskStartedByFilter = "startedBy"
```

Comment:
@dbenhur what are you proposing? I typically see either camel case like this or hyphenated. I'm okay with either way, but we should be consistent. I don't see all lowercase without hyphens very often, if that is your recommendation.